balancer/rls/balancer.go

```diff
@@ -284,6 +286,14 @@ func (b *rlsBalancer) UpdateClientConnState(ccs balancer.ClientConnState) error
 	b.updateCh.Put(resumePickerUpdates{done: done})
 	b.stateMu.Unlock()
 	<-done
+
+	if resizeCache {
+		// If the new config changes the size of the data cache, we might have to
```

Comment:
"If the new config _reduces_ the size"...?

Reply:
Done.